pkg/odo/cli/project/create.go

```diff
@@ -77,20 +77,24 @@ func (pco *ProjectCreateOptions) Run(cmd *cobra.Command) (err error) {
 		return err
 	}
 	s.End(true)
+
+	successMessage := fmt.Sprintf(`Project '%s' is ready for use`, pco.projectName)
```

Comment:
```suggestion
	successMessage := fmt.Sprintf(`Project %q is ready for use`, pco.projectName)
```

Reply:
In general, we're trying to use `%q` wherever we need quotes around something. There are still a bunch of places where we don't show quotes when it's necessary. But we're trying to gradually fix it whenever we notice it in PRs.